pvlib/test/test_pvsystem.py

```diff
@@ -342,18 +318,23 @@ def test_sapm_effective_irradiance(sapm_module_params, test_input, expected):
         assert_allclose(out, expected, atol=1e-4)
 
 
-def test_PVSystem_sapm_effective_irradiance(sapm_module_params):
+def test_PVSystem_sapm_effective_irradiance(sapm_module_params, mocker):
     system = pvsystem.PVSystem(module_parameters=sapm_module_params)
+    mocker.spy(pvsystem, 'sapm_effective_irradiance')
 
-    poa_direct = np.array([np.nan, 1000, 1000])
-    poa_diffuse = np.array([100, np.nan, 100])
-    airmass_absolute = np.array([1.1, 1.1, 1.1])
-    aoi = np.array([10, 10, 10])
-    reference_irradiance = 1000
+    poa_direct = 1000
+    poa_diffuse = 100
+    airmass_absolute = 1.1
+    aoi = 10
+    reference_irradiance = 1100
 
     out = system.sapm_effective_irradiance(
         poa_direct, poa_diffuse, airmass_absolute,
         aoi, reference_irradiance=reference_irradiance)
+    pvsystem.sapm_effective_irradiance.assert_called_once_with(
+        poa_direct, poa_diffuse, airmass_absolute, aoi, sapm_module_params,
+        reference_irradiance=reference_irradiance)
+    assert_allclose(out, 1, atol=0.1)
```

Comment:
This low-precision test will fail if we change the units of effective irradiance. That could be an argument for or against having any low-precision tests at all, depending on your perspective.

Reply:
Here, we are testing that `pvsystem.sapm_effective_irradiance` is called and returns a reasonable value, correct?  If so, then I would change `airmass_absolute = 1.5`, `aoi = 0` and `poa_direct = 900`.  We should keep `reference_irradiance = 1000`. With these inputs the method call should return 1.0 exactly.  At least until we act on #472

Reply:
I updated the input parameters. 

>Here, we are testing that pvsystem.sapm_effective_irradiance is called and returns a reasonable value, correct? 

Yes, so I kept ``atol=0.1``.